# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qtcore/hotfix-qdbus.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qtopia-core-opensource-src-4.3.0rc1/projects.pro.orig	2007-05-06 18:22:28.000000000 +0200
+++ qtopia-core-opensource-src-4.3.0rc1/projects.pro	2007-05-10 12:22:26.000000000 +0200
@@ -26,6 +26,8 @@
     isEqual(PROJECT, tools) {
        !cross_compile:SUBDIRS += tools
        else:SUBDIRS += tools/qtestlib
+    } else:isEqual(PROJECT, tools/qdbus) {
+       SUBDIRS += tools/qdbus
     } else:isEqual(PROJECT, examples) {
        SUBDIRS += examples
     } else:isEqual(PROJECT, demos) {
